pkg/virt-controller/watch/application.go

```diff
@@ -104,6 +104,8 @@ const (
 	exporterImage       = "virt-exportserver"
 	launcherQemuTimeout = 240
 
+	migrationControllerRestTimeout = 30 * time.Second
```

Comment:
Isn't this too much?

Reply:
It is a long timeout, but this is a big change and I wanted to make absolutely sure I don't break anything...
Rest calls usually take less than a second, maybe a couple on a busy cluster.
However, when running operations on hundreds of VMs at a time, I wouldn't be surprised with delays in the 2 digits.
I feel like 30 should definitely be safe, but if you and others feel like it's too much, 20 or even 15 should be fine.